test/Test/Pos/Ssc/GodTossing/ComputeSharesSpec.hs

```diff
@@ -29,8 +32,22 @@ spec = describe "computeSharesDistr" $ do
     prop totalStakeZeroDesc totalStakeIsZero
     prop validRichmenStakesWorksDesc validRichmenStakesWorks
     prop lrcConsistencyDesc lrcConsistency
+
+    describe "distributionIsLimited" $ do
```

Comment:
By _I think it makes more sense to use regular English words here instead of something that reminds identifiers in code because these are not identifiers._ I mean that it makes sense to change `distributionIsLimited` to `Distribution is limited`.